packages/components/backtop/__tests__/backtop.spec.tsx

```diff
@@ -3,7 +3,7 @@ import { mount } from '@vue/test-utils'
 import { CaretTop } from '@element-plus/icons-vue'
 import Backtop from '../src/backtop.vue'
 
-const _mount = (template: VNode) =>
+const _mount = (render: () => VNode) =>
   mount(
     {
       setup: () => () => template,
```

Comment:
I don't understand, since there we are passing a `VNode` into `_mount` function, why are we declaring the parameter as `function`? Besides, this `render` was never used since you did not reference it in this function.

You should do either:

```ts
const _mount = (renderer: () => VNode) => {
  return mount({
     setup() {
        return renderer
     }
  })
}
```

or 

```ts
const _mount = (template: VNode) => {
  return mount(template)
}
```

Reply:
simplified!